### PR TITLE
bump core version to 2.361 and update Jenkinsfile to run on Java 11

### PR DESCRIPTION
**Pull request by olamy**
Signed-off-by: Olivier Lamy <olamy@apache.org>

# Description

See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by jglick** [comment]
Could fold in #146.

File: pom.xml [hunk 1]
```diff
@@ -12,8 +12,7 @@
   <packaging>hpi</packaging>
 
   <properties>
-    <jenkins.version>2.277.4</jenkins.version>
-    <java.level>8</java.level>
```
Comment (jglick) [inline pom.xml:16]: also delete https://github.com/jenkinsci/sse-gateway-plugin/blob/c8f9b702b1d0e7847cb8c3b437de501586b1c34e/pom.xml#L118-L128 for clarity

**Review by jglick — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -73,13 +74,12 @@
     <dependency>
       <groupId>javax.servlet</groupId>
       <artifactId>javax.servlet-api</artifactId>
```
Comment (jglick) [inline pom.xml:76]: I think this dep could simply be deleted—implied for any plugin.

Comment (olamy) [reply pom.xml:76]: ah yup. I reckon I might be a bit scrupulous and always declare in a pom the dependencies really used :)

**Review by jglick — APPROVED** [review]

**Comment by jglick** [comment]
I do not think I specifically need this released, but if you do release it, please let me know.

**Comment by olamy** [comment]
> I do not think I specifically need this released, but if you do release it, please let me know.

I'm on it now and looking at other PRs (dependabot) then I will do it.
A release every 6 months doesn't hurt much

**Pull request merged**